docs/index.rst

```diff
@@ -101,9 +92,7 @@ Office hours
    :hidden:
 
    install/python.rst
-   learn-more/vscode/index.rst
```

Comment:
why did you delete this entry?

Reply:
My bad, that is an artifact from an experiment that wasn't supposed to be included in this pull.